projects/smart-table/src/public-api.ts

```diff
@@ -2,6 +2,6 @@
  * Public API Surface of smart-table
  */
 
-export * from './lib/smart-table/smart-table.service';
-export * from './lib/smart-table/smart-table.component';
+export * from './lib/services/api.service';
```

Comment:
De types worden niet mee geexporteerd en kunnen dus extern niet gebruikt worden. Volgens mij is het beter om ze wel te exporteren?